src/widget/wlibrarysidebar.cpp

```diff
@@ -345,6 +389,29 @@ void WLibrarySidebar::selectChildIndex(const QModelIndex& index, bool selectItem
     scrollTo(translated, EnsureVisible);
 }
 
+QModelIndex WLibrarySidebar::selectedIndex() {
+    QModelIndexList selectedIndices = selectionModel()->selectedRows();
+    if (selectedIndices.isEmpty()) {
+        return QModelIndex();
+    }
+    QModelIndex selIndex = selectedIndices.first();
+    VERIFY_OR_DEBUG_ASSERT(selIndex.isValid()) {
+        return QModelIndex();
+    }
```

Comment:
QModelIndex() is also invalid anyway. 
```suggestion
    DEBUG_ASSERT(selIndex.isValid());
```

Reply:
:+1: